engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactory.scala

```diff
@@ -47,20 +51,33 @@ class KafkaTypedAvroSourceFactory[T: TypeInformation](schemaRegistryProvider: Sc
              )
              @NotBlank
              topic: String,
-             // json or avro schema on this level?
              @ParamName("schema")
              @NotBlank
-             avroSchema: String): Source[T] with TestDataGenerator = {
+             avroSchema: String): Source[T] with TestDataGenerator =
+    createKafkaAvroSource(
+      processMetaData,
+      topic,
+      new FixedKafkaAvroSchemaProvider(topic, avroSchema, kafkaConfig, formatKey, useSpecificAvroReader)
```

Comment:
what error will we show on UI when schema is not correct? We should try better than show "Fatal validation error" or sth like that. If we want to do it in next PR, let's add some TODO?

Reply:
I will add TODO with:
- Avro Validator BE / FE 
- Editor ACE with JSON mode